<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imgicons/icones/Gamepad.png);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Usuário</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_usuario">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>Digite seu usuário</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_senha">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Digite sua senha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_cadastrar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img-icons.qrc"/>
 </resources>
 <connections/>
</ui>
